db/model/roomType.js

```diff
@@ -123,19 +123,21 @@ module.exports = function roomType(seq, dataTypes) {
             resolve(inst);
           }
 
-          inst._previousDataValues.bots.forEach((botName) => {
-            seq.models.Bot.findOne({
-              where: {
-                name: {
-                  $iLike: botName,
+          if (inst._previousDataValues.bots) {
+            inst._previousDataValues.bots.forEach((botName) => {
```

Comment:
@iamigo This is something that I will need to change as it may not be safe. Thanks.